protocol/CMakeLists.txt

```diff
@@ -88,6 +88,51 @@ install(
 
 SET_TARGET_PROPERTIES(${PROJECT_NAME} PROPERTIES VERSION ${ILM_API_VERSION} SOVERSION ${ILM_API_VERSION})
 
+if (IVI_SHARE)
+    project (ivi-share)
+
+    add_custom_command(
+        OUTPUT  ivi-share-client-protocol.h
+        COMMAND ${WAYLAND_SCANNER_EXECUTABLE} client-header
+                < ${CMAKE_SOURCE_DIR}/protocol/ivi-share.xml
+                > ${CMAKE_CURRENT_BINARY_DIR}/ivi-share-client-protocol.h
+        DEPENDS ${CMAKE_SOURCE_DIR}/protocol/ivi-share.xml
+    )
+
+    add_custom_command(
+        OUTPUT  ivi-share-server-protocol.h
+        COMMAND ${WAYLAND_SCANNER_EXECUTABLE} server-header
+                < ${CMAKE_SOURCE_DIR}/protocol/ivi-share.xml
+                > ${CMAKE_CURRENT_BINARY_DIR}/ivi-share-server-protocol.h
+        DEPENDS ${CMAKE_SOURCE_DIR}/protocol/ivi-share.xml
+    )
```

Comment:
you do not need to generate server header file. Please remove it

Reply:
I removed and pushed it.